tests/gpflow/experimental/check_shapes/test_inheritance.py

```diff
@@ -180,6 +183,50 @@ def f(self, a: TensorType) -> TensorType:
                 return t(1)
 
 
+@pytest.mark.parametrize("enable_super", [True, False])
+@pytest.mark.parametrize("enable_sub", [True, False])
+def test_inherit_check_shapes__disable(enable_super: bool, enable_sub: bool) -> None:
```

Comment:
Could you add a docstring that explains what you want to test here?

Reply:
> Could you add a docstring that explains what you want to test here?

Sure.